app/pages/organization/organization-container.jsx

```diff
@@ -41,8 +40,8 @@ class OrganizationContainer extends React.Component {
 
   componentWillReceiveProps(nextProps, nextContext) {
     const { user } = this.context;
-    const { location, params } = this.props;
-    const { collaboratorView, fetchingOrganization, organization } = this.state;
```

Comment:
In the above constructor, why does `this.state.collaboratorView` default to `true`?
When looking at the Snapshot Org, I am seeing `collaborator: false` while `collaboratorView: true`.